core/new-gui/src/environments/environment.default.ts

```diff
@@ -90,14 +95,6 @@ export const defaultEnvironment = {
     accessToken: "",
   },
 
-  /**
-   * all google-related configs
-   */
-  google: {
-    clientID: "",
-    publicKey: "",
-  },
-
```

Comment:
Make sure you alert developers and service maintainers about the breaking change in the config.

Reply:
Sure.